ndc-reference/bin/reference/main.rs

```diff
@@ -103,12 +128,14 @@ fn init_app_state() -> AppState {
     // Read the CSV data files
     let articles = read_csv("articles.csv").unwrap();
     let authors = read_csv("authors.csv").unwrap();
+    let universities = read_json_lines("universities.json").unwrap();
```

Comment:
Can we use CSV like for everything else? Otherwise we can switch the others to JSON but I think it would be nice to have one format.

Reply:
JSON is certainly more convenient for nested structures than CSV. I'll change the other files to JSON.

Reply:
Done